proto/tendermint/liquidity/v1beta1/tx.proto

```diff
@@ -0,0 +1,207 @@
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "\"1\"",
+      format: "uint32"
+  }];
+
+  // reserve coin pair of the pool to deposit
+  repeated cosmos.base.v1beta1.Coin deposit_coins = 4 [(gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"deposit_coins\"",
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "[{\"denom\": \"denomX\", \"amount\": \"1000000\"}, {\"denom\": \"denomY\", \"amount\": \"2000000\"}]",
+      format: "sdk.Coins"
+    }];
+}
+
+// MsgCreatePoolResponse defines the Msg/CreatePool response type.
+message MsgCreatePoolResponse {}
+
+// `MsgDepositWithinBatch defines` an `sdk.Msg` type that supports submitting deposit request to the batch of the liquidity pool
+// Deposit submit to the batch of the Liquidity pool with the specified `pool_id`, `deposit_coins` for reserve
+// this requests are stacked in the batch of the liquidity pool, not immediately processed and
+// processed in the `endblock` at once with other requests.
+//
+// See: https://github.com/tendermint/liquidity/blob/develop/x/liquidity/spec/04_messages.md
+message MsgDepositWithinBatch {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string depositor_address = 1 [(gogoproto.moretags) = "yaml:\"depositor_address\"",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "account address of the origin of this message",
+      example: "\"cosmos1e35y69rhrt7y4yce5l5u73sjnxu0l33wvznyun\"",
+      format: "sdk.AccAddress"
+    }];
+
+  // id of the target pool
+  uint64 pool_id = 2 [(gogoproto.moretags) = "yaml:\"pool_id\"", (gogoproto.jsontag) = "pool_id",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "\"1\"",
+      format: "uint64"
+    }];
+
+  // reserve coin pair of the pool to deposit
+  repeated cosmos.base.v1beta1.Coin deposit_coins = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"deposit_coins\"",
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "[{\"denom\": \"denomX\", \"amount\": \"1000000\"}, {\"denom\": \"denomY\", \"amount\": \"2000000\"}]",
+      format: "sdk.Coins"
+    }];
+
+}
+
+// MsgDepositWithinBatchResponse defines the Msg/DepositWithinBatch response type.
+message MsgDepositWithinBatchResponse {}
+
+// `MsgWithdrawWithinBatch` defines an `sdk.Msg` type that supports submitting withdraw request to the batch of the liquidity pool
+// Withdraw submit to the batch from the Liquidity pool with the specified `pool_id`, `pool_coin` of the pool
+// this requests are stacked in the batch of the liquidity pool, not immediately processed and
+// processed in the `endblock` at once with other requests.
```

Comment:
```suggestion
// processed in the `endblock` at the same time as other requests.
```

Reply:
some of these suggestions are repetitive... I am now losing track of where the edits are in place, some of this looks familiar. Can we merge all for this round and open new PRs moving forward?

Reply:
Sure, Let me apply and merge all for this round

Reply:
@barriebyron I just apply the suggestions from https://github.com/tendermint/liquidity/pull/222, 
And open new PR https://github.com/tendermint/liquidity/pull/222 based latest branch